guess_number.rb

```diff
@@ -1 +1,15 @@
-# guess the number!
+my_num = rand(1..10)
+guess = 0
+puts "The nubmer is #{my_num}"
+puts "Guess the number!"
+while my_num != guess
+guess = gets.chomp
+guess = guess.to_i
```

Comment:
you can call the to_i directly on the gets.chomp, no need for two lines. not wrong to do it this way though